data/changelog.txt

```diff
@@ -789,6 +816,13 @@ Make farm plots re-tillable in farming extension version 2
 Break fake vehicle parts when the real part breaks
 Fixes OM Terrains triggering their exit_eoc on entry instead of correctly triggering their entry_eoc
 Raw copper wire (weighing 4.747 g) can no longer be disassembled into copper (5 g). New recipe to turn raw copper wire into copper.
+Stop birdshot from firing 20 projectiles, each of which has the power of 100 projectiles
+Travel to no longer cancels on moving up/down on a ramp
+[Bugfixes] Fix concentration metamagic sign flipping
+Offspring inherit friendly value from parents to prevent conflict with tamed parents
+Updates trapfunc.cpp to consider quadrupedal movement
+LIXA teleport death bugfix
+[MoM] Fix Prescient Tactics learning
```

Comment:
```suggestion
Stop birdshot from firing 20 projectiles, each of which has the power of 100 projectiles
Travel to no longer cancels on moving up/down on a ramp
Offspring inherit friendly value from parents to prevent conflict with tamed parents
LIXA teleport death bugfix
```